clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/v3client/RegionScopedV3TitusClient.java

```diff
@@ -67,6 +71,9 @@
 
   private final JobManagementServiceGrpc.JobManagementServiceBlockingStub grpcBlockingStub;
 
+  @Autowired
```

Comment:
Can we move this into the constructor ... this class is constructor created so the `@Autowired` stands out as strange.